--- include/mgl2/abstract.h	2018-01-19 14:24:28.257691800 +0100
+++ new_abstract.h	2018-01-19 14:21:14.057656100 +0100
@@ -203,7 +203,7 @@
 /// Callback function for asking user a question. Result shouldn't exceed 1024.
 extern MGL_EXPORT void (*mgl_ask_func)(const wchar_t *quest, wchar_t *res);
 /// Console function for asking user a question. Result shouldn't exceed 1024.
-void mgl_ask_gets(const wchar_t *quest, wchar_t *res);
+void MGL_EXPORT mgl_ask_gets(const wchar_t *quest, wchar_t *res);
 /// Callback function for displaying progress of something.
 extern MGL_EXPORT void (*mgl_progress_func)(int value, int maximal, HMGL gr);
 /// Console function for displaying progress of something.
