--- CMakeLists.txt	2019-10-29 10:52:18.507507100 +0100
+++ New_CMakeLists.txt	2019-10-29 10:51:59.303243000 +0100
@@ -1,6 +1,15 @@
 cmake_minimum_required(VERSION 3.1.0)
 
-project( MathGL2 )
+project( MathGL2 C CXX )
+include(${CMAKE_BINARY_DIR}/../conan_paths.cmake)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_POSITION_INDEPENDENT_CODE:BOOL ON)
+if (APPLE)
+    set(CMAKE_INSTALL_RPATH "@executable_path/../lib")
+else()
+    set(CMAKE_INSTALL_RPATH "$ORIGIN/../lib")
+endif()
 
 set(mgl_clean_files )
 set(MGL_DEP_LIBS)
@@ -14,7 +23,7 @@
 		None(CMAKE_CXX_FLAGS or CMAKE_C_FLAGS used) Debug Release RelWithDebInfo MinSizeRel." FORCE)
 endif(NOT CMAKE_BUILD_TYPE)
 
-set(CMAKE_VERBOSE_MAKEFILE ON)
+set(CMAKE_VERBOSE_MAKEFILE OFF)
 set(MathGL_VERSION_MAJOR 2)
 set(MathGL_VERSION_MINOR 4)
 set(MathGL_PATCH_VERSION 4)
@@ -535,16 +544,9 @@
 	if(NOT MGL_HAVE_PNG)
 		message(SEND_ERROR "You have to enable PNG if you plan to use PDF export.")
 	endif(NOT MGL_HAVE_PNG)
-	find_library(HPDF_LIB hpdf)
-	if(NOT HPDF_LIB)
-		message(SEND_ERROR "Couldn't find libHaru or libhpdf.")
-	endif(NOT HPDF_LIB)
-	find_path(HPDF_INCLUDE_DIR hpdf_u3d.h)
-	if(NOT HPDF_INCLUDE_DIR)
-		message(SEND_ERROR "Couldn't find headers of 3d-enabled version of libhpdf.")
-	endif(NOT HPDF_INCLUDE_DIR)
-	include_directories(${HPDF_INCLUDE_DIR})
-	set(MGL_DEP_LIBS ${HPDF_LIB} ${MGL_DEP_LIBS})
+        find_package(libharu REQUIRED)
+
+        set(MGL_DEP_LIBS libharu::libharu ${MGL_DEP_LIBS})
 else(enable-pdf)
 	set(MGL_HAVE_PDF 0)
 endif(enable-pdf)
