--- widgets/fltk.cpp	2017-11-28 19:38:20.913309000 +0100
+++ fltk.cpp	2018-01-21 18:37:46.929247000 +0100
@@ -107,7 +107,13 @@
 	tet_val = phi_val = 0;
 	draw_par = 0;	draw_func = 0;	draw_cl = 0;
 	last_id = -1;	run = false;
-	popup=0;	vpar=0;	wpar=0;	thr=0;
+	popup=0;	vpar=0;	wpar=0;	
+#if (MGL_HAVE_PTHREAD|MGL_HAVE_PTHR_WIDGET)
+	thr=0;
+#else
+	use_pthr = false;
+#endif
+
 }
 //-----------------------------------------------------------------------------
 Fl_MathGL::~Fl_MathGL()	{	if(mgl_use_graph(gr,-1)<1)	mgl_delete_graph(gr);	}
